### PR TITLE
tracecmd & gantt: format & combine campaigns

### PR DESCRIPTION
**Pull request by apaolillo**

### CODE REVIEW

**Review by matsvanmolle — COMMENTED** [review]
Please address the one comment.

File: benchkit/charts/gant.py [hunk 1]
```diff
@@ -176,75 +181,103 @@ def parse_file(filename, first_pid=-1):
...
+                                "cpu": parsed_line["cpu"],
+                                "pid": parsed_line["pid"],
+                            }
+                        )
+                    elif parsed_line["type"] == "sched_switch" and (
+                        parsed_line["prev_pid"] in exec_pids or parsed_line["next_pid"] in exec_pids
+                    ):
+                        cpu_usage[parsed_line["next_pid"]] = parsed_line[
+                            "cpu"
+                        ]  ######This could be wrong
```
[the remark this answers is not in the document]
Comment (apaolillo) [reply benchkit/charts/gant.py:210]:
```suggestion
```

**Comment by apaolillo** [comment]
done! Are we good to go?

**Comment by matsvanmolle** [comment]
> done! Are we good to go?

yes!

**Pull request merged**